packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -170,13 +170,15 @@ export default function SidebarNavigationScreenGlobalStyles() {
 	const { setCanvasMode, setEditorCanvasContainerView } = unlock(
 		useDispatch( editSiteStore )
 	);
-
-	const isStyleBookOpened = useSelect(
-		( select ) =>
-			'style-book' ===
-			unlock( select( editSiteStore ) ).getEditorCanvasContainerView(),
-		[]
-	);
+	const { isViewMode, isStyleBookOpened } = useSelect( ( select ) => {
+		const { getCanvasMode, getEditorCanvasContainerView } = unlock(
+			select( editSiteStore )
+		);
+		return {
+			isViewMode: getCanvasMode() === 'view',
```

Comment:
Nit (and totally optional) - the expression below uses Yoda condition. For neatness, should give them the same style? Either one or the other, I don't believe there's a preferred way. 😄